src/mbgl/gl/drawable_gl.cpp

```diff
@@ -108,7 +108,7 @@ void DrawableGL::bindUniformBuffers() const {
             const auto& block = uniformBlocks.get(id);
             if (!block) continue;
             const auto& uniformBuffer = getUniformBuffers().get(id);
-            assert(uniformBuffer && "UBO missing, drawable skipped");
+            /*assert(uniformBuffer && "UBO missing, drawable skipped");
```

Comment:
do we want / can we still do this check?

Reply:
No, because now some UBO can miss at the drawable level, being set a layer group level. So I will remove this assert.